x-pack/plugins/security_solution_serverless/public/navigation/navigation_tree/navigation_tree.ts

```diff
@@ -163,7 +163,6 @@ const formatFooterNodesFromLinks = (
           title: category.label,
           icon: category.iconType,
           breadcrumbStatus: 'hidden',
-          defaultIsCollapsed: true,
```

Comment:
This is required to have the "auto-expand" accordion (based on the URL location) working.